packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -3195,6 +3195,12 @@ interface Disposable_2 {
 }
 export { Disposable_2 as Disposable }
 
+// @public
+export interface DtmfDialPadOptions {
+    // (undocumented)
+    dialerBehavior?: 'autoShow' | 'alwaysShow' | 'alwaysHide';
```

Comment:
Will this auto show the dialer button, or the dialpad in the video gallery?

Reply:
This controls the gallery view, the user will always see the dialer in the gallery if its always show, and hidden if always hide. if its auto it uses our existing logic